### ISSUE TITLE
Include expandable stack-trace for actions, in the "Inspector" tab

### ISSUE THREAD

**Issue by Venryx** [reduxjs/redux-devtools#368]
This is a feature proposal, similar to: https://github.com/evgenyrodionov/redux-logger/issues/158

**Comment by Venryx** [comment]
For others wanting this behavior, this is the workaround I'm currently using:
```
	const store = createStore(
		[...]
		compose(
			[...]
			applyMiddleware([
				store=>next=>action=> {
					// add stack, so we can inspect in redux-devtools
					action["stack"] = new Error().stack.split("\n");
					const returnValue = next(action);
					return returnValue;
				},
			]),
		)
	);
```

The result (make sure you open the "Action" panel, rather than the "Diff" panel):
![](http://i.imgur.com/f5Dqila.png)

**Comment by zalmoxisus** [comment]
We finally implemented that in [`2.17.0`](https://github.com/zalmoxisus/redux-devtools-extension/releases/tag/2.17.0). Thanks for the suggestion!
